controller.js

```diff
@@ -45,4 +46,18 @@ exports.getAllArticles = (req, res, next)=>{
         res.status(200).send({articles})
     })
     .catch(next)
+}
+
+exports.postArticleComment = (req, res, next)=>{
+
+const article_id = req.params.article_id
+const username = req.body.username
+const commentBody = req.body.body
+
+    Promise.all([checkArticleExists(article_id),createArticleComment(article_id, username, commentBody)])
+    .then((newComment)=>{
```

Comment:
Be careful with naming your variables. This is slightly confusing (to say the least) as newComment is _not_ a new comment it is the array returned by Promise.all. You could destructure the new comment and then proceed (see MY next comment 😄 ).